recipes/backup_client.rb

```diff
@@ -17,10 +18,27 @@
 template "#{node['stash']['home_path']}/backup-config.properties" do
   source 'backup-config.properties.erb'
   owner node['stash']['user']
-  mode '0644'
-  variables :backup_client => settings['backup_client']
+  mode '0600'
+  variables :backup => settings['backup']
 end
 
 link "#{node['stash']['backup_client']['install_path']}/stash-backup-client/backup-config.properties" do
   to "#{node['stash']['home_path']}/backup-config.properties"
 end
+
+directory node.set['stash']['backup']['backup_path'] do
+  owner node['stash']['user']
+  mode '0750'
```

Comment:
The backup archive is using `stash` as the owner. I think even in `/tmp` where others can see the archive file but they cannot open it. We are still safe enough. 

I think guarding `/tmp` from changing permission is :+1: